File: CPP/Kode Program/Aksesoris.cpp
#include <iostream>
#include <string>
#include "PetShop.cpp"

using namespace std;

class Aksesoris : public PetShop
{
    private:
       string Jenis;
       string Bahan;
       string Warna;

    public:
        // Konstruktor
        Aksesoris() 
        {

        }

        // Setter dan Getter Jenis
        void setJenis(string jenis)
        {
            Jenis = jenis;
        }

        string getJenis()
        {
            return Jenis;
        }

        // Setter dan Getter Bahan
        void setBahan(string bahan)
        {
            Bahan = bahan;
        }

        string getBahan()
        {
            return Bahan;
        }

        // Setter dan Getter Warna
        void setWarna(string warna)
        {
            Warna = warna;
        }

        string getWarna()
        {
            return Warna;
        }

        // Destruktor
        ~Aksesoris()
        {
            
        }
};

File: Java/Kode Program/Aksesoris.java

class Aksesoris extends Petshop
{
    
    private String jenis;
    private String bahan;
    private String warna;

    public Aksesoris()
    {
        // Konstruktor default kosong
    }

    public void setJenis(String jenis) 
    {
        this.jenis = jenis;
    }

    public String getJenis() 
    {
        return jenis;
    }

    public void setBahan(String bahan) 
    {
        this.bahan = bahan;
    }

    public String getBahan() 
    {
        return bahan;
    }

    public void setWarna(String warna) 
    {
        this.warna = warna;
    }

    public String getWarna() 
    {
        return warna;
    }
}
